Content.Server/GameObjects/Components/Items/Storage/StorageCounterComponent.cs

```diff
@@ -25,6 +25,7 @@ namespace Content.Server.GameObjects.Components.Items.Storage
     [RegisterComponent]
     public class StorageCounterComponent : Component, ISerializationHooks
     {
+        // TODO Convert to EntityWhitelist
```

Comment:
probably wont do this in this PR but leaving it here since it would make sense for StorageCounter to have an EntityWhitelist and not just do it based on tags